chia/_tests/cmds/test_farm_cmd.py

```diff
@@ -46,7 +46,14 @@ async def receiver_available() -> bool:
     wallet_rpc_port = wallet_service.rpc_server.webserver.listen_port
     farmer_rpc_port = farmer_service.rpc_server.webserver.listen_port
 
-    await summary(full_node_rpc_port, wallet_rpc_port, None, farmer_rpc_port, bt.root_path)
+    await summary(
+        rpc_port=full_node_rpc_port,
+        wallet_rpc_port=wallet_rpc_port,
+        harvester_rpc_port=None,
+        farmer_rpc_port=farmer_rpc_port,
+        include_pool_rewards=False,
```

Comment:
Hi Will, can you make this True so that the CI coverage runs are more complete. I know it won't exercise the plot NFT but it will get the prints.

Reply:
@wjblanke done

Reply:
I think that may have mucked up the tests, didn't realize it was matching the prints

FAILED .venv/lib/python3.10/site-packages/chia/_tests/cmds/test_farm_cmd.py::test_farm_summary_command[ConsensusMode.HARD_FORK_2_0] - AssertionError: assert 'Block rewards:' in 'Farmer rewards: 0.0'

Reply:
Ah, I forgot to check back to see if CI was passing. I think I know what you are after though. I'll set it back to False, and try to get some testing added that executes the code I've added